paasta_tools/autoscale_cluster.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# Copyright 2015 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import argparse
+import logging
+
+from paasta_tools.autoscaling_lib import autoscale_local_cluster
+
+
+log = logging.getLogger('__main__')
```

Comment:
ah, yeah, it's because the ``addHandler`` function returns None.  try this:

```
log = logging.getLogger(__name__)
log.addHandler(logging.NullHandler())

log.debug('foo')
```
I think we probably should do it everywhere that is a library, to protect against the caller not configuring logging - [the docs](https://docs.python.org/2/howto/logging.html#configuring-logging-for-a-library) mention that a message will be printed to stderr if they haven't and the library tries to log.